### DISCUSSION TITLE
How to avoid gitlens extend the width of current window?

### DISCUSSION THREAD

**Discussion by weqopy** [gitkraken/vscode-gitlens#1722] [Q&A]
There is no scroll bar at bottom normally like this:
<img width="669" alt="image" src="https://user-images.githubusercontent.com/10494196/141764488-d570f67d-a164-415e-be4a-916877aad5a1.png">
After moving cursor to another line is long enough:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/10494196/141764627-1d37e09d-8b89-4db5-acbf-b079b9120a40.png">

Any way to avoid this situation?

**Comment by eamodio** [comment]
Can you ensure that `"gitlens.currentLine.scrollable"` is set to `true` in your `settings.json` or is missing altogether?

**Reply by weqopy** [reply]
> Can you ensure that `"gitlens.currentLine.scrollable"` is set to `true` in your `settings.json` or is missing altogether?

Yes, this setting is used and I can scroll right to see the whole message.
However, my question is this scroll extend the default width of the window, I have to scroll to see the whole message.
<img width="655" alt="image" src="https://user-images.githubusercontent.com/10494196/141921847-192373f9-eaea-4856-8155-adb5484958be.png">

**Reply by eamodio** [reply]
Are you asking to show the git information below the current line? If so, that is problematic because you would not want it overlaid on top of actual code.

**Reply by weqopy** [reply]
This is only a first thought about to avoid my actual question.
I don't know if there is already some settings can implement it.
